habitat_baselines/rl/models/simple_cnn.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+import torch
+import torch.nn as nn
+
+from habitat_baselines.rl.ppo.utils import Flatten
+
+
+class SimpleCNN(nn.Module):
+    def __init__(self, observation_space, output_size):
+        super().__init__()
+        if "rgb" in observation_space.spaces:
+            self._n_input_rgb = observation_space.spaces["rgb"].shape[2]
+        else:
+            self._n_input_rgb = 0
+
+        if "depth" in observation_space.spaces:
+            self._n_input_depth = observation_space.spaces["depth"].shape[2]
+        else:
+            self._n_input_depth = 0
+
+        # kernel size for different CNN layers
+        self._cnn_layers_kernel_size = [(8, 8), (4, 4), (3, 3)]
```

Comment:
Maybe worth to make method arguments with default values.

Reply:
The logic is hard coded to the length of this list.